cockroachcloud/architecture.md

```diff
@@ -62,7 +62,7 @@ Finally, the SQL pods communicate with the KV layer to access data managed by th
 
 #### Baseline
 
-Baseline performance for a Serverless cluster is 100 [Request Units](learn-about-request-units.html), or RUs, per second, and any usage above that is called [burst performance](#cockroachdb-cloud-terms). Clusters start with 10M RUs of free burst capacity each month and earn 100 RUs per second up to a maximum of 250M free RUs per month. Earned RUs can be used immediately or accumulated as burst capacity. If you use all of your burst capacity, your cluster will revert to baseline performance.
+Baseline performance for a Serverless cluster is 100 [Request Units](learn-about-request-units.html), or RUs, per second, and any usage above that is called [burst performance](#cockroachdb-cloud-terms). Clusters start with 10M RUs of free burst capacity each month and earn 100 RUs per second up to a maximum of 50M free RUs per month. Earned RUs can be used immediately or accumulated as burst capacity. If you use all of your burst capacity, your cluster will revert to baseline performance.
```

Comment:
Note that we won't switch to 50M free RUs per month until June 1st. I guess the idea is that we'll underpromise but over-deliver until then?

Reply:
@marksoper can you confirm this is the plan for now?

Reply:
Yes, this was the guidance we got from legal.